addons/mail/static/src/core_ui/message.xml

```diff
@@ -108,7 +108,7 @@
                                                 <t t-else="">
                                                     <em t-if="message.subject and !message.isSubjectSimilarToOriginThreadName and !message.isSubjectDefault" class="mb-1 me-2">Subject: <t t-esc="message.subject"/></em>
                                                     <t t-if="message.body" t-out="message.body"/>
-                                                    <small t-if="message.editDate" class="ms-2 fst-italic text-muted" t-att-title="message.editDatetimeHuge">(edited)</small>
+                                                    <small t-if="message.editDate" class="o-mail-Message-edited ms-2 fst-italic text-muted" t-att-title="message.editDatetimeHuge">(edited)</small>
```

Comment:
While we're at it. I noticed a strange display of the (edited) label on empty messages (e.g. edited attachment only messages). This is due to the `ms-2` on the small tag. Could we fix that at the same time?